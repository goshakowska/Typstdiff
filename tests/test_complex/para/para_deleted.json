{"pandoc-api-version":[1,23,1],"meta":{},"blocks":[{"t":"Para","c":[{"t":"Str","c":"In"},{"t":"Space"},{"t":"Str","c":"this"},{"t":"Space"},{"t":"Str","c":"report,"},{"t":"Space"},{"t":"Str","c":"we"},{"t":"Space"},{"t":"Str","c":"will"},{"t":"Space"},{"t":"Str","c":"explore"},{"t":"Space"},{"t":"Str","c":"the"},{"t":"SoftBreak"},{"t":"Str","c":"various"},{"t":"Space"},{"t":"Str","c":"factors"},{"t":"Space"},{"t":"Str","c":"that"},{"t":"Space"},{"t":"Str","c":"influence"},{"t":"Space"},{"t":"Str","c":"structures."}]},{"t":"Para","c":[{"t":"Str","c":"It"},{"t":"Space"},{"t":"Str","c":"is"},{"t":"Space"},{"t":"Str","c":"the"},{"t":"Space"},{"t":"Str","c":"responsibility"},{"t":"Space"},{"t":"Str","c":"of"},{"t":"Space"},{"t":"Str","c":"the"},{"t":"Space"},{"t":"Str","c":"author"},{"t":"Space"},{"t":"Str","c":"to"},{"t":"Space"},{"t":"Str","c":"obtain"},{"t":"Space"},{"t":"Str","c":"any"},{"t":"Space"},{"t":"Str","c":"required"},{"t":"Space"},{"t":"Str","c":"government"},{"t":"Space"},{"t":"Str","c":"or"},{"t":"Space"},{"t":"Str","c":"company"},{"t":"Space"},{"t":"Str","c":"reviews"},{"t":"Space"},{"t":"Str","c":"for"},{"t":"Space"},{"t":"Str","c":"their"},{"t":"Space"},{"t":"Str","c":"papers"},{"t":"Space"},{"t":"Str","c":"in"},{"t":"Space"},{"t":"Str","c":"advance"},{"t":"Space"},{"t":"Str","c":"of"},{"t":"Space"},{"t":"Str","c":"publication."},{"t":"Space"},{"t":"Str","c":"Start"},{"t":"Space"},{"t":"Str","c":"early"},{"t":"Space"},{"t":"Str","c":"to"},{"t":"Space"},{"t":"Str","c":"determine"},{"t":"Space"},{"t":"Str","c":"if"},{"t":"Space"},{"t":"Str","c":"the"},{"t":"Space"},{"t":"Str","c":"reviews"},{"t":"Space"},{"t":"Str","c":"are"},{"t":"Space"},{"t":"Str","c":"required;"},{"t":"Space"},{"t":"Str","c":"this"},{"t":"Space"},{"t":"Str","c":"process"},{"t":"Space"},{"t":"Str","c":"can"},{"t":"Space"},{"t":"Str","c":"take"},{"t":"Space"},{"t":"Str","c":"several"},{"t":"Space"},{"t":"Str","c":"weeks."}]}]}
