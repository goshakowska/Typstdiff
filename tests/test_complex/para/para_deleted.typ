In this report, we will explore the
various factors that influence structures.

It is the responsibility of the author to obtain any required government or company reviews for their papers in advance of publication. Start early to determine if the reviews are required; this process can take several weeks.
