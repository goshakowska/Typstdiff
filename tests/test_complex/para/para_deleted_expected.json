{"pandoc-api-version": [1, 23, 1], "meta": {}, "blocks": [{"t": "Para", "c": [{"t": "Str", "c": "In"}, {"t": "Space"}, {"t": "Str", "c": "this"}, {"t": "Space"}, {"t": "Str", "c": "report,"}, {"t": "Space"}, {"t": "Str", "c": "we"}, {"t": "Space"}, {"t": "Str", "c": "will"}, {"t": "Space"}, {"t": "Str", "c": "explore"}, {"t": "Space"}, {"t": "Str", "c": "the"}, {"t": "SoftBreak"}, {"t": "Str", "c": "various"}, {"t": "Space"}, {"t": "Str", "c": "factors"}, {"t": "Space"}, {"t": "Str", "c": "that"}, {"t": "Space"}, {"t": "Str", "c": "influence"}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Emph", "c": [{"t": "Str", "c": "fluid"}, {"t": "SoftBreak"}, {"t": "Str", "c": "dynamics"}]}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "in"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "glaciers"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "ha"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "they"}]}, {"t": "Strikeout", "c": [{"t": "SoftBreak"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "contribute"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "to"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "the"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "formation"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "and"}]}, {"t": "Strikeout", "c": [{"t": "SoftBreak"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "behaviour"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "of"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "these"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "natural"}]}, {"t": "Space"}, {"t": "Str", "c": "structures."}]}, {"t": "Para", "c": [{"t": "Strikeout", "c": [{"t": "Str", "c": "All"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "manuscripts"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "are"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "to"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "be"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "submitted"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "electronically"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "to"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "the"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "ScholarOne"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "Abstracts"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "site"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "created"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "for"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "each"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "conference."}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "The"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "manuscript"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "upload"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "will"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "be"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "enabled"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "several"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "weeks"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "after"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "acceptance"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "notices"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "have"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "been"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "sent."}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "Presenting"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "authors"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "of"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "accepted"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "papers"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "will"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "receive"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "an"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "email"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "with"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "instructions"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "when"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "manuscript"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "submission"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "opens."}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "It"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "is"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "important"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "that"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "presenting"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "authors"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "keep"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "their"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "email"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "addresses"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "up-to-date"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "so"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "they"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "do"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "not"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "miss"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "this"}]}, {"t": "Strikeout", "c": [{"t": "Space"}]}, {"t": "Strikeout", "c": [{"t": "Str", "c": "notice."}]}]}, {"t": "Para", "c": [{"t": "Str", "c": "It"}, {"t": "Space"}, {"t": "Str", "c": "is"}, {"t": "Space"}, {"t": "Str", "c": "the"}, {"t": "Space"}, {"t": "Str", "c": "responsibility"}, {"t": "Space"}, {"t": "Str", "c": "of"}, {"t": "Space"}, {"t": "Str", "c": "the"}, {"t": "Space"}, {"t": "Str", "c": "author"}, {"t": "Space"}, {"t": "Str", "c": "to"}, {"t": "Space"}, {"t": "Str", "c": "obtain"}, {"t": "Space"}, {"t": "Str", "c": "any"}, {"t": "Space"}, {"t": "Str", "c": "required"}, {"t": "Space"}, {"t": "Str", "c": "government"}, {"t": "Space"}, {"t": "Str", "c": "or"}, {"t": "Space"}, {"t": "Str", "c": "company"}, {"t": "Space"}, {"t": "Str", "c": "reviews"}, {"t": "Space"}, {"t": "Str", "c": "for"}, {"t": "Space"}, {"t": "Str", "c": "their"}, {"t": "Space"}, {"t": "Str", "c": "papers"}, {"t": "Space"}, {"t": "Str", "c": "in"}, {"t": "Space"}, {"t": "Str", "c": "advance"}, {"t": "Space"}, {"t": "Str", "c": "of"}, {"t": "Space"}, {"t": "Str", "c": "publication."}, {"t": "Space"}, {"t": "Str", "c": "Start"}, {"t": "Space"}, {"t": "Str", "c": "early"}, {"t": "Space"}, {"t": "Str", "c": "to"}, {"t": "Space"}, {"t": "Str", "c": "determine"}, {"t": "Space"}, {"t": "Str", "c": "if"}, {"t": "Space"}, {"t": "Str", "c": "the"}, {"t": "Space"}, {"t": "Str", "c": "reviews"}, {"t": "Space"}, {"t": "Str", "c": "are"}, {"t": "Space"}, {"t": "Str", "c": "required;"}, {"t": "Space"}, {"t": "Str", "c": "this"}, {"t": "Space"}, {"t": "Str", "c": "process"}, {"t": "Space"}, {"t": "Str", "c": "can"}, {"t": "Space"}, {"t": "Str", "c": "take"}, {"t": "Space"}, {"t": "Str", "c": "several"}, {"t": "Space"}, {"t": "Str", "c": "weeks."}]}]}